### ISSUE TITLE
[Question] Will the project see any further development? 

### ISSUE THREAD

**Issue by Tisckkytt** [Tunous/WebMark#4]
If yes, I'll create some issues with basic but important features like Import/Export, tagging / some type of organizational solution at basic level etc.

**Comment by Tunous** [comment]
Unless someone else will contribute here then (at this time) it's unlikely. I've switched my main device to iOS so it's no longer used by me.

I experimented sometime ago with adding synchronization features and iOS app so they could both work together. But right now it's more of a rough version. Can't say about the future, I might continue on the above idea which would make me develop this Android version further. No promises though.

In short there is 20% chance for yes and 80% chance for no.
